### PR TITLE
fix: modify build scripts

### PR DESCRIPTION
**Pull request by JivusAyrus**
#### Checklist

- [ ] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: examples/vite-swr/.wundergraph/wundergraph.config.ts [hunk 1]
```diff
@@ -30,7 +30,14 @@ configureWunderGraphApplication({
 	],
 	cors: {
 		...cors.allowAll,
-		allowedOrigins: ['http://localhost:5173'],
+		allowedOrigins: process.env.NODE_ENV === 'production' ? ['https://*'] : ['http://*'],
```
Comment (StarpTech) [inline examples/vite-swr/.wundergraph/wundergraph.config.ts:33]: That's confusing. Does it work without specifying `allowedOrigins` at all?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: examples/vite-swr/.wundergraph/wundergraph.config.ts (hunk 1 above)
Comment (JivusAyrus) [reply examples/vite-swr/.wundergraph/wundergraph.config.ts:33]: no, allowedOrigins is a required attribute

Comment (StarpTech) [reply examples/vite-swr/.wundergraph/wundergraph.config.ts:33]: So using `	...cors.allowAll,` alone does not allow all? :smile:

**Review by StarpTech — APPROVED** [review]
LGTM

**Review by jensneuse — APPROVED** [review]

File: packages/sdk/src/cors/index.ts [hunk 2]
```diff
@@ -1,18 +1,10 @@
...
-	allowedOrigins: [
-		{
-			kind: ConfigurationVariableKind.STATIC_CONFIGURATION_VARIABLE,
-			staticVariableContent: '*',
-			environmentVariableName: '',
-			environmentVariableDefaultValue: '',
-			placeholderVariableName: '',
-		},
-	],
+	allowedOrigins: ['http://*', 'https://*'],
```
Comment (StarpTech) [inline packages/sdk/src/cors/index.ts:7]: You mean `["*"]`?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: packages/sdk/src/cors/index.ts (hunk 2 above)
Comment (JivusAyrus) [reply packages/sdk/src/cors/index.ts:7]: That doesnt work.

**Review by fiam — APPROVED** [review]
LGTM

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**